### ISSUE TITLE
Section “13. Accessibility Support” unclear

### ISSUE THREAD

**Issue by nitedog** [w3c/wcag-act#221]
From [Stefan Schnabel](https://lists.w3.org/Archives/Public/public-wcag-act-comments/2018Jul/0000.html):

> Section “13. Accessibility Support”:
>  
> “Because of this, ACT Rules are not necessarily applicable in all test scenarios. For instance, a web page that has to work in assistive technologies that have no WAI-ARIA [WAI-ARIA] support, wouldn’t be tested with an ACT Rule that relies on WAI-ARIA support, since this could lead to false positive results.”
>  
> This is not understandable without further examples or rewording especially what “false positive results” would mean.
> 
> Concealment does not improve things.
> 
> I mean, shouldn’t the rule set actively *encourage* the USE of WAI-ARIA instead of doing protectionism for older AT and user agents?
> 
> Of course, respective rule messaging should be flagged accordingly.

**Comment by WilcoFiers** [comment]
I've made some minor tweaks to it. One of the goals of the rules format was to allow rule designers to express limitations on accessibility support. Website owners are free to choose if they wish to push their users to the most modern assistive technologies out there, or if they wish to hold up some degree of backward compatibility. Rules should not be limiting in that respect.

**Comment by nitedog** [comment]
@WilcoFiers I do not think your edits help remove the negativity. In fact, the sentence now has a jarring double negation that makes it difficult for me to understand it.

More importantly, I do not understand what requirements this section sets and what to include in it. It is also a confusing mix of things that relate to rule authors and rule implementations.

Below is a suggested attempt to rewrite that section, to hopefully make it clearer. I believe it does not change any of the meanings but separates out the different thoughts:

[[
Content can be designed to rely on the support for particular accessibility features by different assistive technologies and user agents. For example, content using a particular WAI-ARIA [[WAI-ARIA]] feature relies on that feature to be supported by assistive technologies and user agents. This content would not work for assistive technologies and user agents that do not support WAI-ARIA. WCAG [[WCAG]] provides a definition for [accessibility supported](https://www.w3.org/TR/WCAG20/#accessibility-supporteddef) use of a Web technology.

ACT Rules, including [=composite rule=] and [=atomic rules=], MUST specify known limitations on accessibility support. For example, an atomic rule that checks for a particular accessibility feature that is known to be not widely supported by assistive technologies and web browsers, or a composite rule that includes atomic rules with known accessibility support limitations.

Implementations of ACT Rules SHOULD convey corresponding notifications to their users when results from rules with known accessibility support limitations are conveyed.

NOTE: WCAG Evaluation Methodology (WCAG-EM) provides guidance on defining an [accessibility support baseline](https://www.w3.org/TR/WCAG-EM/#step1c) for test scenarios, which can help users of ACT Rules to select the appropriate rules for their own circumstance.
]]
